### PR TITLE
feature(Plugins/Premium Plugins/Export): Update docs for the Export p…

### PR DESCRIPTION
**Pull request by hamza0867**
…remium plugin page

Related Ticket: TINY-8595

Description of Changes:
This PR fixes the docs of the Export premium plugin page. This PR tries to be as contained as possible and not cause conflict with other PRs, so the changes are made to the files being rendered only in the /plugins/premium/export.html url
Worth noting that some xrefs might be broken (e.g: the one that link to configure/something.adoc ) as well as some Jekyll syntax that did not get update because it is taken care of in other PRs.

Pre-checks:
- [x] Branch prefixed with `feature/` or `hotfix/`
- [x] `_data/nav.yml` has been updated (if applicable)
- [x] Files has been included where required (if applicable)
- [x] Files removed have been deleted, not just excluded from the build (if applicable)
- [x] (New product features only) Release Note added

Review:
- [x] Documentation Team Lead has reviewed
- [x] Product Manager has reviewed

### CODE REVIEW

File: modules/ROOT/pages/plugins/premium/export.adoc [hunk 1]
```diff
@@ -6,23 +6,24 @@
...
+* Click the Export toolbar button (image:icons/export.svg[Export icon: A page with an arrow from the center of the page to the right of the page]) and select _PDF_.
 
 liveDemo::export[]
 
 == Basic setup
 
-To add the {pluginname} plugin to the editor, add `+{plugincode}+` to the `plugins` option in the editor configuration.
+To add the {pluginname} plugin to the editor, add `{plugincode}` to the `plugins` option in the editor configuration.
 
 For example:
```
Comment (lnewson) [inline modules/ROOT/pages/plugins/premium/export.adoc:29]: Can't leave a comment on the exact line, but the source code block below uses attributes so it needs `subs="attributes+"` added.

File: modules/ROOT/pages/plugins/premium/export.adoc [hunk 2]
```diff
@@ -6,23 +6,24 @@
 :pluginname: Export
 :plugincode: export
 
-include::partial$misc/requires_5_5v.adoc[] +
+include::partial$misc/requires_5_5v.adoc[]
+
 include::partial$misc/premiumplugin.adoc[]
```
Comment (lnewson) [inline modules/ROOT/pages/plugins/premium/export.adoc:11]: Just confirming that the content in this include has been fixed elsewhere? Asking as it's rendering a link incorrectly atm:
image

**Review by lnewson — CHANGES_REQUESTED** [review]
There's a file that appear to have been missed here. export_ignore_elements.adoc` has an issue with the code block as it needs `subs="attributes+"` added for `{plugincode}` to resolve.

File: modules/ROOT/pages/plugins/premium/export.adoc (hunk 2 above)
Comment (hamza0867) [reply modules/ROOT/pages/plugins/premium/export.adoc:11]: Yes it has been fixed in another PR

**Comment by hamza0867** [comment]
> There's a file that appear to have been missed here. export_ignore_elements.adoc`has an issue with the code block as it needs`subs="attributes+"`added for`{plugincode}` to resolve.

Done

File: modules/ROOT/pages/plugins/premium/export.adoc (hunk 1 above)
Comment (hamza0867) [reply modules/ROOT/pages/plugins/premium/export.adoc:29]: Done

**Review by lnewson — APPROVED** [review]

File: modules/ROOT/partials/configuration/image_cors_hosts.adoc [hunk 3]
```diff
@@ -1,23 +1,23 @@
-ifeval::[{plugincode} == "export"]
+ifeval::["{plugincode}" == "export"]
 === `export_cors_hosts`
```
Comment (lnewson) [inline modules/ROOT/partials/configuration/image_cors_hosts.adoc:2]:
```suggestion
[[export_cors_hosts]]
=== `export_cors_hosts`
```

File: modules/ROOT/partials/configuration/image_cors_hosts.adoc [hunk 4]
```diff
@@ -1,23 +1,23 @@
...
 endif::[]
-ifeval::[{plugincode} != "export"]
+ifeval::["{plugincode}" != "export"]
 === `imagetools_cors_hosts`
```
Comment (lnewson) [inline modules/ROOT/partials/configuration/image_cors_hosts.adoc:5]:
```suggestion
[[imagetools_cors_hosts]]
=== `imagetools_cors_hosts`
```

File: modules/ROOT/partials/configuration/image_proxy.adoc [hunk 5]
```diff
@@ -1,21 +1,22 @@
-ifeval::[{plugincode} == "export"]
+ifeval::["{plugincode}" == "export"]
 :proxy_setting_name: export_image_proxy
+[[export_image_proxy]]
 === `export_image_proxy`
 endif::[]
-ifeval::[{plugincode} != "export"]
+ifeval::["{plugincode}" != "export"]
 :proxy_setting_name: imagetools_proxy
 === `imagetools_proxy`
```
Comment (lnewson) [inline modules/ROOT/partials/configuration/image_proxy.adoc:8]:
```suggestion
[[imagetools_proxy]]
=== `imagetools_proxy`
```

**Review by ltrouton — APPROVED** [review]

**Review by EkimChau — APPROVED** [review]

**Pull request merged**